core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -19,8 +19,14 @@
  * you change or add Alluxio configuration properties.
  */
 public enum PropertyKey {
+  CONF_DIRECTORY(Name.CONF_DIRECTORY, "${alluxio.home}/conf"),
   DEBUG(Name.DEBUG, false),
+  /**
+   * @deprecated use INSTALL_HOME or WORK_DIRECTORY instead
+   */
+  @Deprecated
```

Comment:
the problem is that if we keep in as `alluxio.home` we would be changing the meaning of the configuration property. I also prefer `alluxio.home` to `alluxio.install.home`, but don't think we should make a sudden change here without deprecation.

From your other comment about directory vs folder vs home, would you prefer `alluxio.directory`?